App/Theme.ts

```diff
@@ -32,22 +56,98 @@ export const Colors: ColorTheme = {
   toastSuccess: BaseColors.green,
   toastError: BaseColors.red,
   toastInfo: BaseColors.black,
+  accent: BaseColors.yellow,
+  borderLight: '#2E8540',
```

Comment:
Can we add the hex codes to `BaseColors` and reference them here?

Reply:
Will do, you looking for `BaseColors.lightBlue`?

Reply:
Fixed.